css/style.css

```diff
@@ -1,7 +1,355 @@
+/* ----------- VISUALIZATION ----------- */
+
 .circle {
   display: inline-block;
   width: 100px;
   height: 100px;
   border-radius: 50px;
   background-color: red;
 }
+
+/* ----------- UI - General ----------- */
+
+*::before, *::after, * {
+  box-sizing: border-box;
+}
+
+body {
+  display: grid;
+  grid-template-columns: 170px auto;
+  height: 100vh;
+  margin: 0;
+  font-family: "Open Sans", sans-serif;
+  font-weight: normal; 
+}
+
+h1, h2, h3 {
+  margin: 0;
+  font-weight: 400;
+}
+
+h1 {
+  font-size: 1.75em;
+}
+
+h2 {
+  font-size: .75em;
+  font-weight: 700;
+  text-transform: uppercase;
+}
+
+h3 {
+  font-size: 1.2em;
+  color: #73A4B8;
+  text-transform: uppercase;
+}
+
+button {
+  position: relative;
+  top: 0;
+  left: 0;
+  width: 100%;
+  margin-top: 10px;
+  text-align: left;
+  text-indent: calc(10px + 15px);
+  padding: 10px;
+  font-size: .75em;
+  border: none;
+  border-width: 1px;
+  border-radius: 3px;
+  cursor: pointer;
+  color: #EAEAEA;
+  box-shadow: 0 3px 3px #12181B;
+}
+
+.icon--button {
+  position: absolute;
+  top: 50%;
+  left: 10px;
+  transform: translateY(-50%);
+}
+
+.icon {
+  display: inline-block;
+  width: 15px;
+  height: 15px;
+  margin-right: 10px;
+  fill: #EAEAEA;
+}
+
+a {
+  color: #73A4B8;
+  font-size: .75em;
+}
+
+/* ----------- UI - Side Panel ----------- */
+
+aside {
+  background-color: #404850;
+  padding: 15px;
+}
+
+aside h1 {
+  color: #fff;
+}
+
+.logo {
+  max-width: 100%;
+}
+
+aside h2 {
+  margin-top: 20px;
+  color: #EAEAEA;
+}
+
+.nav__button {
```

Comment:
We have mixed use of BEM and non BEM it seems here which is adding code bloat from what I can see.

For some of the code modifiers like `.active` are being used elsewhere `--filter` for modifiers.

For scoping there is `.nav__button` but also `.graph h2`.

I'm really not a fan of BEM which is a coding style choice which perhaps should be discussed but certainly mixing the two for what seems like similar use cases should be avoided.

For the complexity of this CSS it's likely BEM can easily be avoided and more semantic tags could be used like:
```
nav > button
```